### PR TITLE
add foods table to foods.html populated using js

### PR DESCRIPTION
**Pull request by cmacaulay**
Note: It says so many files have changed because a) master was missing a bunch of files for some reason b) I tried to add the compiled css into the gitignore but i had already pushed so it's still there b) I renamed the css folder stylesheets so it says all of those files changed too. 
- the main actual changes were to foods.js (new file) and foods.html (which had been deleted)
🎉 🎉 
- starts the javascript file
- makes the first ajax call to the 'api/foods/' end point to retrieve foods
- saves foods to local storage
- populates table from local storage
📖 📚 
- I found this front end class on local storage really helpful: [http://frontend.turing.io/lessons/json-and-localstorage.html](url)
🛑 
- right now it is making an ajax call to local host, once production code is updated to remove CORS restrictions we should be able to swap in that link
❓ ❔ 
- what should we name our heroku app? 
- it wasn't letting me use ES6 syntax - is it for backend only? OR I was just using incorrect syntax...

### CODE REVIEW

File: foods.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```
Comment (cmacaulay) [inline foods.html:1]: it had deleted this file from master somehow.

File: main.styl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@require '/public/stylesheets/src/*'
```
Comment (cmacaulay) [inline main.styl:1]: it had deleted this too!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
```
Comment (TheKaterTot) [inline index.html:1]: thanks for adding this file back in. We must have accidentally deleted it. Let's make sure we're checking our git status before we make commits.

File: main.styl (hunk 2 above)
Comment (TheKaterTot) [reply main.styl:1]: Now we're back!

File: variables.styl [hunk 4]
```diff
@@ -0,0 +1,15 @@
+sea-blue = rgba(135, 241, 255, 1)
```
Comment (TheKaterTot) [inline variables.styl:1]: Let's make sure this file is nested with all of the other styl files.

**Comment by TheKaterTot** [comment]
We'll update production and get the connection happening ASAP. I'm not sure how ES6 applies to the client-side JS, but we can research. Great work!

**Pull request merged**